src/ui.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+ui::fzf_cmd() {
+  [ "$NAVI_ENV" == "test" ] && echo fzf_mock && return
+  command_exists sk && echo sk && return
+  echo "fzf -i"
```

Comment:
untested, kinda hacky

Reply:
I'm not sure I got it... You're changing the behavior for `fzf` here? Doesn't it work as-is?

Reply:
Actually its the opposite - i'm removing "-i" from skim, but keeping it in `fzf`.